src/periphery/MarketLens.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+// Uncomment this line to use console.log
+
+import "interfaces/IBentoBoxV1.sol";
+import "interfaces/ICauldronV2.sol";
+import {IERC20, IStrictERC20} from "BoringSolidity/interfaces/IERC20.sol";
+import "utils/CauldronLib.sol";
+
+contract MarketLens {
+    address payable public owner;
```

Comment:
doesn't need an owner, and if you need it, you can inherit BoringOwnable or Operatable (ownable + operators permission level)